src/components/organisms/UiList/stories/ConditionWithPrimaryButton.vue

```diff
@@ -0,0 +1,94 @@
+<template>
+  <UiList
+    v-bind="args"
+    class="condition-with-button"
+  >
+    <template
+      v-for="{ name } in args.items"
+      #[name]="item"
+      :key="name"
+    >
+      <div class="condition-with-button__content">
+        <div class="condition-with-button__probability">
+          <UiProgress
+            :value="item?.evidence?.value"
+            :max="10"
+            class="condition-with-button__progress"
+          />
+          <UiText class="ui-text--body-2-comfortable ui-text--theme-secondary condition-with-button__label">
+            {{ item?.evidence?.label }}
+          </UiText>
+        </div>
+        <UiHeading level="3">
+          {{ item?.label }}
+        </UiHeading>
+        <UiText class="ui-text--body-2-comfortable ui-text--theme-secondary condition-with-button__description">
+          {{ item?.description }}
+        </UiText>
+        <UiButton class="ui-button--text ui-button--small">
+          Show details
+        </UiButton>
+      </div>
+    </template>
+    <template #suffix="{ suffixAttrs }">
+      <UiButton v-bind="suffixAttrs">
+        Learn more
+      </UiButton>
+    </template>
+  </UiList>
+</template>
+
+<script setup>
+import {
+  computed,
+  useAttrs,
+  defineOptions,
+} from 'vue';
+import {
+  UiList,
+  UiHeading,
+  UiProgress,
+  UiText,
+  UiButton,
+} from '@infermedica/component-library';
+
+defineOptions({ inheritAttrs: false });
+const attrs = useAttrs();
+const args = computed(() => (attrs));
```

Comment:
Q: Is it really reasonable to use `computed` with an `attrs`? I'm not the Vue guru as you are, but I have doubts regarding these two lines... 🤔  I thought the `computed` property makes sense mostly for reactive variables, like `ref` and `reactive`... Is the `useAttrs()` an exception?
Also, I've found [this GH issue](https://github.com/vuejs/core/discussions/8421) on the topic, may be useful ;)

Reply:
Well, now that I've checked, I'm pretty sure we don't need those two lines because `attrs` is already _kinda reactive_ in the sense that it is a proxy, so it doesn't have to be wrapped by a computed :) 

Also, I have another question regarding the `attrs` / `args` variable. Why do we have `defineOptions({ inheritAttrs: false });` and then after that the `useAttrs()` line? As I can see in this component - the `args` is bound to the most-outer element - shouldn't it inherit those attrs anyway? Or am I getting something wrong?

Reply:
Both address certain issues in Storybook 🐒 related to passing arguments. We have several decorators passing arguments to stories. Occasionally, we need to pass something beyond component props to stories. That's when we opt for useAttrs over props. The ./stories/.*vue should demonstrate how to tailor CL components for specific outcomes to developers. These customizations will be employed in MGP, but you'll only receive style and template.

Reply:
Honestly, I don't see the connection between your response and my comment... 🙈 😅  

I didn't mention anything about props. Props are okay, and attrs are okay. They both are used for different purposes, period. My point was that we don't have to wrap `attrs` with `computed` as this is already a proxy and reflects changes to the DOM. And that's what I learned from the link I posted, and I strongly recommend going deeper into it, as those guys seem to be really smart eggheads :)

And the second comment I posted was about a point of adding `defineOptions({ inheritAttrs: false });` and then passing attrs into the most-outer element in the DOM - doesn't it work like that like always?

Reply:
OK, I remember now everything to help you. I want the recurring elements in the stories to be the same. And
* Why do we have a `computed` `args`? Because sometimes (i hope soo) we will filter something out and stop binding to the component. i.e. when you 'content' for stories you won't bind it to the component. As you see we want to use `v-bind="args"` to pass everything to component. 
* Why `defineOptions({ inheritAttrs: false });` because :top: sometimes we want filter reundant vaules and ... some stories has a few components like UiButton where we bind args to few UiButtons. When we use `defineOptions({ inheritAttrs: false });` we have more controls. 

Stories isn't the production code (especially) script section. These are just examples where sometimes you will need to hack something for the storybook.

Reply:
Ok, then why should stories determine how the Component should look? If some solution is redundant (like wrapping attrs with `computed`) and we don't ever need it in the real-world use cases, then, IMO it's not a good pattern to write the code specifically more complicated only for the stories' purposes. But that's my opinion. I'm nitpicking here, so I'll give an approval :)